packages/calypso-e2e/src/lib/components/me/me-sidebar-component.ts

```diff
@@ -0,0 +1,29 @@
+import { Page } from 'playwright';
+
+/**
+ * Represents the sidebar component on /me endpoint.
+ */
+export class MeSidebarComponent {
+	private page: Page;
+
+	/**
+	 * Constructs an instance of the component.
+	 *
+	 * @param {Page} page The underlying page.
+	 */
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	/**
+	 * Given a string, navigate to the menu on the sidebar.
+	 *
+	 * @param {string} menu Menu item to navigate to.
+	 */
+	async navigate( menu: string ): Promise< void > {
+		await Promise.all( [
+			this.page.waitForNavigation(),
+			this.page.click( `a:has(span:has-text("${ menu }") )` ),
```

Comment:
Optional suggestion: what do you think trying to lock this selector down to the sidebar by including a parent element of the sidebar? I wonder if "a link with the text in it" might be just a little risky for collisions on page content.

Reply:
Moved selector up to an object and made it dynamic, pinned with `.sidebar` as the prefix in bd8bc48bb555cb23660d753d3d43076801065851.